src/theory/booleans/theory_bool_rewriter.h

```diff
@@ -34,7 +34,15 @@ class TheoryBoolRewriter : public TheoryRewriter
   RewriteResponse preRewrite(TNode node) override;
   RewriteResponse postRewrite(TNode node) override;
 
-}; /* class TheoryBoolRewriter */
+ protected:
+  /** Common constants */
+  Node d_true;
+  Node d_false;
+  /** Helper method which performs flattening. */
+  RewriteResponse flattenNode(TNode n, TNode trivialNode, TNode skipNode);
+  /** Helper method for making a negation */
+  Node makeNegation(TNode n);
```

Comment:
Wrong order (data members below functions).
Document parameters/return values of functions.